Santillana-Demo-AR-Experiencia-Residuos/ARSceneViewController.swift

```diff
@@ -35,31 +106,30 @@ class ARSceneViewController: UIViewController {
         let scene = SCNScene()
         
         arSceneView.scene = scene
-        
-        setupPicker()
     }
     
-    override func viewWillAppear(_ animated: Bool) {
-        super.viewWillAppear(animated)
+    //MARk: Observer methods for state of game
+    private func detectingPlanesSetupUI(){
```

Comment:
I don't believe we need private functions here, this View Controller is not used by anyone